scripts/purity_bifs

```diff
@@ -40,10 +40,10 @@ header() {
 
 %% Dialyzer complains of supertypes.
 %-spec is_pure(atom(), atom(), arity()) -> purity:pure() | unknown.
--spec is_known(atom(), atom(), arity()) -> boolean().
+%-spec is_known(atom(), atom(), arity()) -> boolean().
```

Comment:
This was because of `edoc`.

Reply:
Not sure I follow this one, could  you elaborate?

Reply:
Let me see. I think `edoc` was complaining about spec.s with no associated function.

Reply:
I see, one to figure out later then.

Reply:
Sure. I `make src/purity_bifs.erl` and then `rebar3 edoc` (to generate the documentation). I get `purity/src/purity_bifs.erl, function is_pure/3: at line 6: multiple @spec tag.` since the code is now generated as 
```erlang
%-spec is_pure(atom(), atom(), arity()) -> purity:pure() | unknown.
-spec is_known(atom(), atom(), arity()) -> boolean().

%-spec is_pure(atom(), arity()) -> purity:pure() | unknown.
-spec is_known(atom(), arity()) -> boolean().
```

Reply:
One way we could fix this would be to generate the `-spec(_).`s in their proper place, just above the function declaration.